pkg/appconsts/appconsts.go

```diff
@@ -87,6 +87,10 @@ var (
 	// EvidenceNamespaceID is the namespace reserved for evidence.
 	EvidenceNamespaceID = namespace.ID{0, 0, 0, 0, 0, 0, 0, 3}
 
+	// TailTransactionPaddingNamespaceId is the namespace used for padding after
+	// all transactions (ordinary and PFBs) but before blobs.
+	TailTransactionPaddingNamespaceID = namespace.ID{0, 0, 0, 0, 0, 0, 0, 255}
```

Comment:
[non-blocking naming question]
Will we always have only transactions in the reserved namespaces? For example, what happens when we add ISRs? should we change this name then?

Reply:
Agreed. Proposed alternative names:
- `TailReservedNamespacePadding`
- `ReservedNamespacePadding`

Reply:
Created a separate issue b/c this entails two renames:
1. the implementation
2. the specs